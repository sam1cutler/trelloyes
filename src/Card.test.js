import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';

describe('Card component', () => {
    
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Card />, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the Card component as expected for the "e" (fifth) card', () => {
        const tree = renderer
          .create(<Card title='Fifth card' content='lorem ipsum'/>)
          .toJSON();
        expect(tree).toMatchSnapshot();
    });

})