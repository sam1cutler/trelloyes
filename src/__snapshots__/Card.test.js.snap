// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component renders the Card component as expected for the "e" (fifth) card 1`] = `
<div
  className="Card"
>
  <button
    onClick={[Function]}
    type="button"
  >
    delete
  </button>
  <h3>
    Fifth card
  </h3>
  <p>
    lorem ipsum
  </p>
</div>
`;
